caesar_cipher/lib/cipher.rb

```diff
@@ -0,0 +1,28 @@
+class Cipher
+  LOW_ALPHABET = ("a".ord.."z".ord)
+  HIGH_ALPHABET = ("A".ord.."Z".ord)
+  ALPHABET_LENGHT = 26
+  def initialize(rot_key)
+    @rot_key = rot_key
```

Comment:
@zzokki81 generally speaking your class should validate all the input arguments that could prevent it from working correctly. In this case, you should validate `rot_key` here instead of doing it in the main program. Usually, you would validate the arguments and if there's an error you'd raise ArgumentError exception which should be caught by the calling program and handled gracefully.